src/Swarm/TUI/View.hs

```diff
@@ -222,11 +224,7 @@ drawNewGameMenuUI (l :| ls) =
 
   drawDescription :: ScenarioItem -> Widget Name
   drawDescription (SICollection _ _) = txtWrap " "
-  drawDescription (SISingle (s, si)) = do
-    let oneBest = si ^. scenarioBestTime == si ^. scenarioBestTicks
-    let bestRealTime = if oneBest then "best:" else "best real time:"
-    let noSame = if oneBest then const Nothing else Just
-    let lastText = let la = "last:" in padRight (Pad $ T.length bestRealTime - T.length la) (txt la)
+  drawDescription (SISingle (s, si)) =
     vBox . catMaybes $
```

Comment:
The padding logic is getting hairy here; it would probably be cleaner to convert this to a Table widget.